### PR TITLE
LIVE-1004 Hedera integration

### PR DESCRIPTION
**Pull request by mehcode**
https://ledgerhq.atlassian.net/browse/LIVE-1004

Hedera integration in Ledger Live - add accounts / sync / send / receive

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Everything LGTM apart from the `yarn.lock` which should have no diff in theory?

File: package.json [hunk 1]
```diff
@@ -56,39 +54,42 @@
     }
   },
   "dependencies": {
+    "@hashgraph/sdk": "^2.1.1",
```
[the remark this answers is not in the document]
Comment (jackthta) [reply package.json]: I believe it should be gone now 👍🏻

File: src/screens/ReceiveFunds/03-Confirmation.js [hunk 2]
```diff
@@ -157,6 +158,7 @@ export default function ReceiveConfirmation({ navigation, route }: Props) {
     const device = route.params.device;
 
     if (device && !verified) {
+      setAllowNavigation(false);
```
Comment (haammar-ledger) [inline src/screens/ReceiveFunds/03-Confirmation.js:161]: @jackthta Can you please explain this change? Is it mandatory for Hedera?

**Review by ghost — COMMENTED** [review]
Still a bit weird to have a diff in the yarn.lock I think, but probably ok.

File: src/screens/ReceiveFunds/03-Confirmation.js (hunk 2 above)
Comment (jackthta) [reply src/screens/ReceiveFunds/03-Confirmation.js]: My mistake 😅, it probably sneaked through when rebasing. It has been removed 👍🏻.

**Review by ghost — APPROVED** [review]

**Pull request merged**